app/Main.hs

```diff
@@ -6,30 +6,29 @@ module Main
   ) where
 
 import Lorentz
-  (NiceStorage, attachDocCommons, buildMarkdownDoc, def, niceStorageEvi, printLorentzValue,
-  toMichelsonContract, toVal, zeroMutez)
+  (NiceStorage, attachDocCommons, buildMarkdownDoc, def, niceStorageEvi,
+  printLorentzValue, toMichelsonContract, toVal, zeroMutez)
 
 import Data.Aeson.Encode.Pretty (encodePretty, encodePrettyToTextBuilder)
-import Data.ByteString.Lazy.Char8 qualified as BS (putStrLn)
+import qualified Data.ByteString.Lazy.Char8 as BS (putStrLn)
```

Comment:
Uh. Just noticed this. We kinda switched to `ImportQualifiedPost`. I assume this was done by stylish-haskell, but the config is set up with `post_qualify: true`, so this shouldn't have happened. Perhaps your stylish-haskell binary is a bit outdated?

Reply:
Have fixed this.